index.js

```diff
@@ -141,11 +142,13 @@ function storeOption(longOption, optionValue, options, values) {
 const parseArgs = (config = { __proto__: null }) => {
   const args = objectGetOwn(config, 'args') ?? getMainArgs();
   const strict = objectGetOwn(config, 'strict') ?? true;
+  const allowPositionals = objectGetOwn(config, 'allowPositionals') ?? !strict;
```

Comment:
A brief comment on change in previous design/expectations.

This means `parseArgs` may now throw in `strict:false` mode for userArgs for first time. But it is opt-in and I think make sense when write it first time, and makes sense to read later.

Reply:
Yeah, I thought this was better than making it so you could have an explicit `allowPositionals: false` but still get positionals. The alternative is to reject `allowPositionals: false` in combination with `strict: false`, but I didn't see much value in that.